rfc/seo.md

```diff
@@ -0,0 +1,72 @@
+# Gruntwork.io SEO RFC
+
+I read a lot of material in SEO, including going through a [guide to SEO](https://moz.com/beginners-guide-to-seo/).
+
+## Important take aways
+
+- We currently have some broken stuff in our website that should take priority over trying to improve SEO by creating more content. Ranked by (my perceived) importance:
+  - Searching for site:gruntwork.io/guides will show us that our content is duplicated between the www and non www versions of the website. This is diluting our ranking. We need to add a canonical link to our headers
+  - Our posts don't have meta descriptions
+  - Our internal links usually have the same anchor text (the title of the guide), but Google interprets that as not being natural and will penalize us for that
+  - We don't have a [sitemap](https://support.google.com/webmasters/answer/183668?hl=en)
```

Comment:
https://gruntwork.io/sitemap.xml

Reply:
oops!

Reply:
Having a look at it now, it looks like it could be improved.

For example, this doesn't seem like it should be crawled 
```
<url><loc>https://www.gruntwork.io/assets/css/bootstrap/js/tests/visual/carousel.html</loc><lastmod>2018-10-30T22:34:51-07:00</lastmod></url>
```

Reply:
Agreed!